core-field-editors/README.md

```diff
@@ -31,6 +33,17 @@ contentful-extension create --space-id <MY_SPACE_ID>
 ~~~
 
 You can now use the component by following the same steps as with any other
-[UI extension](https://www.contentful.com/blog/2016/07/06/ui-extensions-sdk/).
+[UI extension](ui-ext-guide).
+
+# Available field editors
+
+We currently propose the following field editors:
```

Comment:
“Propose” doesn’t carry the second meaning of the french “proposer” ;). I propose “we currently provide”